### PR TITLE
Adds PR review checklist

### PR DESCRIPTION
**Pull request by oliverroick**
### Proposed changes in this pull request

This PR adds a review checklist to the pull request template. The checklist lists things to look out for. It might not be exhaustive, feel free to add anything that is missing. 

### When should this PR be merged

Anytime

### Risks

None

### Follow up actions

None

### CODE REVIEW

**Comment by oliverroick** [comment]
@amplifi Added comment about migrations

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -29,5 +25,32 @@ migrations, software that we need to install on staging and production
...
--
--
+
+
+### Checklist (for reviewing)
+
+#### General
+
+- [ ] **Is the PR labeled correctly?** It should have the `migration` label if a new migration is added. 
+
```
Comment (linzjax) [inline .github/PULL_REQUEST_TEMPLATE]: **Is this PR explained thoroughly?** Are all the code changes accounted for in the description of the PR?
Maybe something about risk level? If it's a high risk, has this been taken into consideration?

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -29,5 +25,32 @@ migrations, software that we need to install on staging and production
...
+#### Code
+
+- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
+- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests. 
+- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/). 
+
+#### Tests
+
+- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components. If this is a bug fix, there must be a test case for the bug to ensure the issue won’t regress. 
+
```
Comment (linzjax) [inline .github/PULL_REQUEST_TEMPLATE]: I would split bug fix into a separate check, and ask if the new tests break without the new code to fix it.

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (oliverroick) [inline .github/PULL_REQUEST_TEMPLATE]: Good comments, I added those points

**Pull request merged**